update-scripts/nvim-lspconfig/lspconfig-servers.lua

```diff
@@ -0,0 +1,52 @@
+-- This script is heavily inspired by https://github.com/neovim/nvim-lspconfig/blob/master/scripts/docgen.lua
+require("lspconfig")
+local configs = require("lspconfig.configs")
+local util = require("lspconfig.util")
+
+local function require_all_configs()
+	for _, v in ipairs(vim.fn.glob(vim.env.lspconfig .. "/lua/lspconfig/server_configurations/*.lua", 1, 1)) do
+		local module_name = v:gsub(".*/", ""):gsub("%.lua$", "")
+		configs[module_name] = require("lspconfig.server_configurations." .. module_name)
+	end
+end
+
+local function map_list(t, func)
+	local res = {}
+	for i, v in ipairs(t) do
+		local x = func(v, i)
+		if x ~= nil then
+			table.insert(res, x)
+		end
+	end
+	return res
+end
+
+local function sorted_map_table(t, func)
+	local keys = vim.tbl_keys(t)
+	table.sort(keys)
+	return map_list(keys, function(k)
+		return func(k, t[k])
+	end)
+end
+
+require_all_configs()
+
+info = sorted_map_table(configs, function(server_name, server_info)
+	local description = nil
+	if server_info.document_config.docs ~= nil then
+		description = server_info.document_config.docs.description
```

Comment:
Do they provide any other metadata? Or does everything get thrown in here?

Reply:
No there is quite a lot of metadata, like the default values for root patterns, filetypes, or other configuration items.
I went with the minimal amount to replicate what we had

Reply:
Agreed, we can expand on this in future PRs if there's a need.